src/js/core.js

```diff
@@ -1088,6 +1138,66 @@
                 target.innerHTML = html;
                 this.events.updateInput(target, { target: target, currentTarget: target });
             }
+        },
+
+        addElements: function (elements) {
```

Comment:
So we'll also need to do something like what is in `createElementsArray` here, since we need to make sure we do the setup for `<textarea>` elements.  Perhaps we could even try to break up parts of the `setup()` method so that the code paths for setting up the editor and adding elements can be as similar as possible?